go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/longhorn/go-iscsi-helper v0.0.0-20240207150711-0713d1c9f5d1
 	github.com/longhorn/go-spdk-helper v0.0.0-20240219034107-a6d23f96da54
 	github.com/longhorn/longhorn-engine v1.6.0
-	github.com/longhorn/longhorn-instance-manager v1.6.0
+	github.com/longhorn/longhorn-instance-manager v1.6.0-dev-20240105.0.20240219053144-677fbd99cc56
```

Comment:
Need to update IM dep to the right branch.

Reply:
Then this need to be merged first, thanks!
https://github.com/longhorn/longhorn-instance-manager/pull/403

Reply:
Fixed!